DiamondLightSource/hyperion
# Organise util scripts

Fixes #1231 

The wiki will need to be updated to reference the new location of the deploy script.

### To test:
1. Look at how hyperion takes up four less lines in the vscode explorer, and you can see more of dodal
2. Run the `dls_dev_env` script in and out of the hyperion dir (in a new deployment if you want)
3. Test that the deploy script still works (except it won't find the `dls_dev_env`, since it's in a different place now - confirm the location the deploy script looks will be correct)
